gms/database/scripts/index/dataset-bootstrap.sql

```diff
@@ -0,0 +1,88 @@
+/*
+ First, create a two-column table composed of urn & urnParts
+ This table will only contain distinct URNs which only exist in metadata_aspect but not in metadata_index
+ urn: Actual dataset URN
+ urnParts: Stripped dataset URN which only contains fields of the dataset URN
+ Example dataset_urn_parts table:
++-----------------------------------------------------+------------------------------------+
+| urn                                                 | urn_parts                          |
++-----------------------------------------------------+------------------------------------+
+| urn:li:dataset:(urn:li:dataPlatform:ambry,aaa.2,EI) | urn:li:dataPlatform:ambry,aaa.2,EI |
++-----------------------------------------------------+------------------------------------+
+ */
+CREATE TEMPORARY TABLE dataset_urn_parts
+  SELECT urn,
+         substring(urn, 17, urnLength - 17) AS urn_parts # len("urn:li:dataset:(") = 16 and indexing in SQL starts at 1
+  FROM   (SELECT DISTINCT urn,
+                          length(urn) AS urnLength
+          FROM   metadata_aspect
+          WHERE  urn NOT IN (SELECT DISTINCT urn
+                             FROM   metadata_index
+                             WHERE  aspect="com.linkedin.common.urn.DatasetUrn")
+                 AND urn RLIKE "^urn:li:dataset:\\(urn:li:dataPlatform:[^:,]+,[^:,]+,[^:,]+\\)$"
+                 AND aspect="com.linkedin.common.Status"
+                 AND version=0
+                 AND metadata LIKE '%false%')
```

Comment:
+ @keremsahin1 
Do we still want to exclude deleted datasets not to make into SCSI ?  I couldn't recollect our earlier meeting's conclusion on this.

Reply:
We decided to index everything regardless of removal status into the SCSI. I need to change internal bootstrap script.

Reply:
Updated the script to index all urns irrespective of their `removed` flag.